src/workers/notifications/squadPublicRequestNotification.ts

```diff
@@ -0,0 +1,39 @@
+import { messageToJson } from '../worker';
+import { NotificationSourceContext } from '../../notifications';
+import { NotificationType } from '../../notifications/common';
+import { NotificationWorker } from './worker';
+import { ChangeObject } from '../../types';
+import {
+  Source,
+  SquadPublicRequest,
+  SquadPublicRequestStatus,
+} from '../../entity';
+
+interface Data {
+  request: ChangeObject<SquadPublicRequest>;
+}
+
+const statusToTypeMap: Record<SquadPublicRequestStatus, NotificationType> = {
+  [SquadPublicRequestStatus.Approved]: NotificationType.SquadPublicApproved,
+  [SquadPublicRequestStatus.Rejected]: NotificationType.SquadPublicRejected,
+  [SquadPublicRequestStatus.Pending]: NotificationType.SquadPublicSubmitted,
+};
+
+const worker: NotificationWorker = {
+  subscription: 'api.v1.squad-public-request-notification',
+  handler: async (message, con) => {
```

Comment:
Can you add test for this worker please

Reply:
Oh, right. Missed this one. Will add one soon.

Reply:
The tests for this worker were [contained here](https://github.com/dailydotdev/daily-api/pull/1928/files#diff-8fac815be5a5848da5cbaf80e5c7c2773fee6d00a1824fd23b420907df70fb70R1662-R1696), do you want me to put it in a separate file? I noticed notifications workers were jammed in a single file so I went ahead and added lines.